packages/unmock-core/src/generator.ts

```diff
@@ -152,10 +152,17 @@ const generateMockFromTemplate = (
   // First iteration simply parses these and returns the updated schema
   const resolvedTemplate = jsf.generate(template);
   jsf.reset();
+  // After one-pass resolving we might have new parameters to resolve.
+  let body: string;
+  try {
+    body = JSON.stringify(jsf.generate(resolvedTemplate));
+  } catch {
```

Comment:
This happens on textual responses (when trying to `jsf.generate` from non-object).